docs/reference/run.md

```diff
@@ -981,12 +980,11 @@ or two examples of how to pass more parameters to that ENTRYPOINT:
 ## EXPOSE (incoming ports)
 
 The Dockerfile doesn't give much control over networking, only providing
-the `EXPOSE` instruction to give a hint to the operator about what
-incoming ports might provide services. The following options work with
-or override the Dockerfile's exposed defaults:
+the `EXPOSE` instruction to specify incoming ports that provide services.
```

Comment:
The original version is incorrect. The `EXPOSE` instruction does not "give a hint to the operator", it explicitly specifies exposed ports and cannot be overridden. And, none of the "following options" "override the Dockerfile's exposed defaults".